### DISCUSSION TITLE
In ERT, electrodes are zero or 1-based?

### DISCUSSION THREAD

**Discussion by mariosgeo** [gimli-org/pyGIMLi#615] [General]
Hi, 
I am making few data input for ERT inversion. I am not sure anymore, if in the data section(# a,b,m,n) you refer to the electrodes using zero based or 1-based format?
AFAIK, coming from bert, you had 1-based. I see now that when you create scheme in pygimli, are zero-based.
Then, what is the numbering of the remote electrode (0 or -1)?

Thanks

**Comment by halbmy** [comment]
The data format (!) had been introduced long time ago (when we used Matlab) and electrodes started from 1, 0 being infinite electrodes. This is still the case.
When working inside Python, one starts counting from zero, -1 being infinite.

**Comment by mariosgeo** [comment]
Ok, I convert my data to zero bases and the remote electrode -1 (I have a pole-dipole) array.
Pygmili reports that 
     
`Warning: removed 62511 values due to sensor index out of bounds!
     Data validity check: found 62511 invalid data. 
     Data validity check: remove invalid data.
`
Basically it removed all of my data. What am I missing?

**Comment by halbmy** [comment]
In the files the remote needs to be 0!
Only if you compute in Python it is -1, if you save it, it will be 0 again.
